packages/postgres-database/src/simcore_postgres_database/models/projects_comments.py

```diff
@@ -0,0 +1,55 @@
+import sqlalchemy as sa
+from sqlalchemy.sql import func
+
+from .base import metadata
+from .projects import projects
+
+projects_comments = sa.Table(
+    "projects_comments",
+    metadata,
+    sa.Column(
+        "comment_id",
+        sa.BigInteger,
+        nullable=False,
+        autoincrement=True,
+        primary_key=True,
+        doc="Primary key, identifies the comment",
+    ),
+    sa.Column(
+        "project_uuid",
+        sa.String,
+        sa.ForeignKey(
+            projects.c.uuid,
+            name="fk_projects_comments_project_uuid",
+            ondelete="CASCADE",
+            onupdate="CASCADE",
+        ),
+        doc="project reference for this table",
+    ),
+    sa.Column(
+        "user_id",
+        sa.BigInteger,
+        doc="user reference for this table",
+    ),
+    sa.Column(
+        "content",
+        sa.String,
+        nullable=True,
+        doc="Content of the comment",
+    ),
+    sa.Column(
+        "created_at",
```

Comment:
For created_at and updated_at use the functions in https://github.com/ITISFoundation/osparc-simcore/blob/master/packages/postgres-database/src/simcore_postgres_database/models/_common.py

See other tables as examples!